packages/main/cypress/support/commands/TimePicker.commands.ts

```diff
@@ -0,0 +1,88 @@
+Cypress.Commands.add("ui5TimePickerGetInnerInput", { prevSubject: true }, subject => {
+	cy.wrap(subject)
+		.as("timePicker");
+
+	cy.get("@timePicker")
+		.shadow()
+		.find("ui5-datetime-input")
+		.shadow()
+		.find("input")
+		.as("innerInput");
+
+	return cy.get("@innerInput");
+});
+
+Cypress.Commands.add("ui5TimePickerValueHelpIconPress", { prevSubject: true }, subject => {
+	cy.wrap(subject)
+		.as("timePicker");
+
+	cy.get("@timePicker")
+		.shadow()
+		.find("ui5-datetime-input")
+		.find(".ui5-time-picker-input-icon-button")
+		.realClick();
```

Comment:
confirm that the popover is opened after value state icon is clicked to avoid test flakiness using ui5ResponsivePopover commands.